app/src/main/java/com/door43/translationstudio/newui/translate/FirstTabFragment.java

```diff
@@ -118,6 +119,7 @@ public void onConfirmTabsDialog(String targetTranslationId, String[] sourceTrans
                 if(targetTranslation != null) {
                     try {
                         targetTranslation.addSourceTranslation(sourceTranslation);
+                        TargetTranslationMigrator.mergeInvalidChunksFromProject(AppContext.getLibrary(), targetTranslation);
```

Comment:
Limitation of my code.  Had to wait till the user selected a source translation when doing initial import.